cli/deployment/api_client.go

```diff
@@ -208,14 +232,40 @@ func Delete(deployment *Deployment, preview bool) (string, error) {
 	if preview {
 		args = append(args, "--preview")
 	}
-	output, err := runGCloud(args...)
+	output, err := RunGCloud(args...)
 	if err != nil {
 		log.Printf("failed to delete deployment: %v, error: %v, output: %s", deployment, err, output)
 		return output, err
 	}
 	return output, nil
 }
 
+func GetDeploymentDescription(name string, project string) (*DeploymentDescription, error) {
+	args := []string{
+		"deployment-manager",
+		"deployments",
+		"describe",
+		name,
+		"--project",
+		project,
+		"--format", "yaml",
+	}
+	response, err := RunGCloud(args...)
+	if err != nil {
+		return nil, err
+	}
+
+	description := &DeploymentDescription{}
+
+	err = yaml.Unmarshal([]byte(response), description)
+	if err != nil {
+		log.Printf("error unmarshall response: %s,\n deployment: %v \n error: %v", response, name, err)
```

Comment:
```suggestion
		log.Printf("error unmarshalling response: %s,\n deployment: %v \n error: %v", response, name, err)
```

Reply:
I just realized this is phrased the same somewhere else, so please leave this as-is for now and fix it separately.